es6-classes/shape.js

```diff
@@ -0,0 +1,30 @@
+/* exported Shape */
+class Shape {
+  constructor(area, circumference) {
+    this.area = area;
+    this.circumference = circumference;
+  }
+
+  print() {
+    const value = 'Area is ' + this.area + ' Circumference is ' + this.circumference;
+    return value;
+  }
+}
+
+const newShape = new Shape(23, 34);
+console.log(newShape.print());
+
+class Square extends Shape {
+  constructor(area, circumference, width) {
```

Comment:
Per instruction:
Write its constructor to take the **square's width as a parameter** (and nothing else)